.travis.yml

```diff
@@ -1,8 +1,14 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.3.3
+  - 2.2.5
+  - 2.3.1
```

Comment:
Using `2.3.1` here because Travis machines used to have a few Rubies pre-baked until recently, which made for faster builds.
[Turns out](http://rubies.travis-ci.org/) they only boot with 1.9.3 and install the right one now, so I'll change to use the latest patchlevel of each Ruby version.